src/main/java/org/openrewrite/staticanalysis/DefaultComesLastVisitor.java

```diff
@@ -133,33 +133,35 @@ private List<J.Case> maybeReorderFallthroughCases(List<J.Case> cases, P p) {
 
     private List<J.Case> addBreakToLastCase(List<J.Case> cases, P p) {
         return ListUtils.mapLast(cases, e -> {
-            if (isFallthroughCase(e)) {
+            if (e != null && isFallthroughCase(e.getStatements())) {
```

Comment:
`e` cannot be null here, so no need for this check!

---

FYI: The reason why it's shown as nullable is because the mapList second argument is defined as a `UnaryOperator<@Nullable T>`. We already fixed this by adding a secondary mapList function where this argument is listed as `Function<T, @Nullable T>`, but the original function is still there for backwards compatibility.